content/blocks/turrets/scatter.json

```diff
@@ -0,0 +1,10 @@
+ammo: {
+  cobblestone : {
+    type: BasicBulletType
+    damage: 4
+    speed: 3.3
+    lifetime: 36
+    inaccuracy: 9
+    ammoMultiplier: 2
+  }
```

Comment:
You'd have to redefine every other vanilla scatter ammos here, or else scatter will only shoot gravel